### ISSUE TITLE
is this repo done?

### ISSUE THREAD

**Issue by IIIdefconIII** [MarkusMcNugen/docker-qBittorrentvpn#173]
im wondering if this repo is done because im on :latest and...
[5.0.2-r0-ls365](https://github.com/linuxserver/docker-qbittorrent/releases/tag/5.0.2-r0-ls365) [Latest](https://github.com/linuxserver/docker-qbittorrent/releases/latest)

already exist and my docker v is on :latest but...
qBittorrent v4.3.8 Web UI (64-bit)

just wondering if we need to move over to another repo

**Comment by MarkusMcNugen** [comment]
Yes, this is no longer actively maintained. You might be able to find an
updated fork of it if someone else started their own but I no longer have
the time to keep this going and haven't been active for some years now.
